common/test/acceptance/tests/video/test_video_events.py

```diff
@@ -149,6 +151,48 @@ def test_strict_event_format(self):
         assert_events_equal(static_fields_pattern, load_video_event)
 
 
+class VideoHLSEventsTest(VideoEventsTestMixin):
+    """ Test video player event emission for HLS video """
+
+    def setUp(self):
+        """
+        Run test in chrome.
+        """
+        with patch.dict(os.environ, {'SELENIUM_BROWSER': 'chrome'}):
+            super(VideoHLSEventsTest, self).setUp()
+
+    def test_event_data_for_hls(self):
+        """
+        Scenario: Video component with HLS vide emits events correctly
+
+        Given the course has a Video component with Youtube, HTML5 and HLS sources available.
+        And I play the video
+        And the video starts playing
+        And I watch 3 seconds of it
+        Then I pause the video
+        Then I seek the video
+        Then I play the video until it ends
```

Comment:
When I pause and seek the video
And I play the video to the end
Then I verify that all expected events are triggered
And triggered events have correct data